upload_Faces_Batch.py

```diff
@@ -0,0 +1,119 @@
+import requests
+import argparse
+import sys
+import json
+import csv
+import time
+
+class uploadDirectory:
+    #Set up workspace for API calls to Rhombus Systems
+    def __init__(self,args):
+        #Initialize argument parser
+        self.argParser = self.__initArgParser()
+        self.args = self.argParser.parse_args(args)
+        #Create a session to set default call Header information
+        self.session = requests.session()
+        self.session.headers = {"x-auth-scheme": "api-token","x-auth-apikey": self.args.apiKey}
+        #Set a default base URL for api calls
+        self.url = "https://api2.rhombussystems.com/api/"
+
+    #Define arguments which the user may be prompted for
+    @staticmethod
+    def __initArgParser():
+        argParser = argparse.ArgumentParser(description='Upload a selection of faces for facial recognition. List file names in a CSV on individual lines.Please follow the nameing protocal: First-Last_#.jpg')
+        argParser.add_argument("apiKey", help="Get this from your Console.")
+        argParser.add_argument("addr", help="Local Address of Photo Directory CSV")
```

Comment:
As it currently stands, requiring a CSV file seems pretty useless.  I would suggest either allowing a user to just point to a directory containing well-formatted filenames and/or having the provided csv file contain both the person's name and a local filename and having this script make sure that the name of the file uploaded to the server is in the right format.

Reply:
Newest Commit addressees this if I understood correctly:
User points to a directory -> All .jpg files are evaluated for upload based on adherence to naming convention

or

User points to a csv -> each row should contain a file path and a name ("Michael_Phelps-Rio2016.jpg","Mike P") -> face is uploaded with supplied name